js/views/chatview.js

```diff
@@ -242,6 +240,29 @@
 			}
 		},
 
+		_getDateSeparator: function(timestamp) {
+			var date = moment(timestamp, 'x'),
+				today = moment(),
```

Comment:
Maybe it would be better to call it `now` instead of `today`? Not really sure which one would be clearer, though.